gen.py

```diff
@@ -136,19 +137,14 @@ def convert_svg2png(infile, outfile, w, h):
         for size in sizes:
             root = "{0}.svg".format(icon["linux"]["root"])
             source = "icons/{0}/{1}/{2}.svg".format(theme, size, icon_name)
-            output = "{0}/{1}/apps/{2}".format(linux_dir, size, root)
+            output = "{0}/{1}/apps/".format(linux_dir, size)
             if path.exists(source):
-                bfb_icon = icon["linux"].get("bfb")
-                if bfb_icon:
-                    output_bfb = "{0}/{1}/apps/{2}.png".format(linux_dir,
-                                                               size,
-                                                               bfb_icon)
-                    if int(size) == 48:
-                        convert_svg2png(source, output_bfb, 144, 144)
-                copy2(source, output)
+                if icon["linux"].get("bfb") and (size == "48"):
+                    output_bfb = "{0}.png".format(icon["linux"].get("bfb"))
+                    convert_svg2png(source, output+output_bfb, 144, 144)
```

Comment:
There's a missing space between the `+`

Reply:
I was trying to work out why neither my linter nor the GH check had picked that up. Turns out spaces around arithmetic operators [are no longer required](https://hg.python.org/peps/rev/37af28ad2972) in PEP8. However assignments, comparisons, boolean, and curiously `%` still need them though.

Is there a way we can tell the checker here to also check the disabled by default E226?

Reply:
There must be a way, I will take a look at this later :+1: